src/components/Home.vue

```diff
@@ -1,14 +1,130 @@
 <template lang="pug">
   div
-    h1 {{ msg }}
+    h1.heading-1 Vueable Component Library
+    h1.heading-2 Introduction
+    p.
+      Welcome! We're glad you've found us. This little library of components started at a time
+      when there were already a few choices for Vue.js component libraries, but none of them really felt
+      right to us. We wanted something lightweight, easy to get started with, and something that didn't require
+      changes to our existing apps. We also felt like composability should really be favored over configuration
+      and that wasn't as well represented as we had hoped.
+    p.
+      We've looked deeply at Element, Quasar, Blueprint (React) and others in order to get a decent idea about
+      what the community is looking for in a component library and what this library should be about. That said, we
+      are not terribly original in our thinking or comprehensive in our component list. But our hope is that if
+      you need a basic set of components to do simple UIs with then we might be a fit for you.
```

Comment:
*puts on editor hat*

This is a really good start. I like the layout and the concepts presented under headings, so people really get the philosophy behind why Vueable is useful.

I'm not sure about the "we" concept for these first two paragraphs. I would write it in a more declarative style, about the library itself, rather than introducing it as "we made this, here's the story". Something like:

Vueable is a lightweight component library designed to be composable, simple to configure, and easy to integrate into existing apps.

It takes inspiration from Element, Quasar, and Blueprint (React). As a work in progress, it is not comprehensive or even terribly original in its component list. But if you need a basic set of components for simple UIs, Vueable may be a good fit.

Reply:
I have a really terse writing style, so feel free to ignore this. I've been told I'm kind of a jerk for cutting a lot of words :)